both/methods.js

```diff
@@ -185,12 +185,13 @@ Meteor.methods({
     check(competitionId, String);
     throwIfCannotManageCompetition(this.userId, competitionId);
 
-    var round = Rounds.findOne(roundId);
+    var round = Rounds.findOne(roundId); // Will not exist for some events
```

Comment:
Nit/philosophical issue: `roundId` will be null for non WCA events such as lunch, right? Seems strange to me to attempt to look up a Round with a null id. Taking this a step further, I feel like we should assert/throw an exception if we're passed a non-null roundId that doesn't actually correspond to a Round.